keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -294,6 +294,9 @@ class CombinedRepository(
 
   override fun deliveryConfigsDueForCheck(minTimeSinceLastCheck: Duration, limit: Int): Collection<DeliveryConfig> =
     deliveryConfigRepository.itemsDueForCheck(minTimeSinceLastCheck, limit)
+
+  override fun getAll(): Collection<DeliveryConfig> =
+    deliveryConfigRepository.getAll()
```

Comment:
Is it worth disambiguating the type in the method name in case we add a `getAll` for resources, or whatever?